tavern/graphql/gqlgen.yml

```diff
@@ -1,35 +1,28 @@
 schema:
-  - schema/types.graphql      # Common Golang type definitions.
-  - schema/relay.graphql      # Types and Definitions required by Relay.  
-  - schema/query.graphql      # API Query Definitions.
-  - schema/mutation.graphql   # API Mutation Definitions.
-  - schema/credential.graphql # Credential GraphQL Schema.
-  - schema/deployment.graphql # Deployment GraphQL Schema.
-  - schema/file.graphql       # File GraphQL Schema.
-  - schema/tag.graphql        # Tag GraphQL Schema.
-  - schema/target.graphql     # Target GraphQL Schema.
-  - schema/implant.graphql    # Implant GraphQL Schemas.
-  - schema/ent.graphql        # The ent.graphql schema was generated by Ent.
+  - schema/scalars.graphql
+  - schema/schema.graphql
+  - schema/user.graphql
+
+model:
+  package: graphql
+
+exec:
+  layout: follow-schema
+  dir: ./generated
+  package: generated
 
 resolver:
   # Tell gqlgen to generate resolvers next to the schema file.
   layout: follow-schema
   dir: .
+  package: graphql
 
 # gqlgen will search for any type names in the schema in the generated
 # ent package. If they match it will use them, otherwise it will new ones.
 autobind:
   - github.com/kcarretto/realm/tavern/ent
-  - github.com/kcarretto/realm/tavern/ent/credential
-  - github.com/kcarretto/realm/tavern/ent/deployment
-  - github.com/kcarretto/realm/tavern/ent/deploymentconfig
   - github.com/kcarretto/realm/tavern/ent/file
-  - github.com/kcarretto/realm/tavern/ent/implant
-  - github.com/kcarretto/realm/tavern/ent/implantcallbackconfig
-  - github.com/kcarretto/realm/tavern/ent/implantconfig
-  - github.com/kcarretto/realm/tavern/ent/implantserviceconfig
-  - github.com/kcarretto/realm/tavern/ent/tag
-  - github.com/kcarretto/realm/tavern/ent/target
+  - github.com/kcarretto/realm/tavern/ent/user
```

Comment:
Probably just my not understanding ent.go but do we need to include the tome scheme in here?

Reply:
I think it is now, but yeah you're right